simpleui-server/src/simpleui-server.ts

```diff
@@ -60,11 +60,36 @@ export class SimpleUIServer {
         return ipAddress;
     }
 
+    /**
+     * Adds overrides from command line to props by directly manipulating the
+     * props object
+     * @param cmdVars
+     * @param props
+     */
+    static addOverrides(cmdVars: CommandArgs, props: any): any {
+        if (cmdVars.theme) {
+            Logger.log(LogLevel.INFO, `Overriding props.appTheme.name (${props.appTheme.name}) with ${cmdVars.theme}`);
+            props.appTheme.name = cmdVars.theme;
+        }
+
+        if (cmdVars.DBname) {
+            let macros = props.macro;
+            macros.forEach( (macro) => {
+                if (macro.token.includes("_MYSQL_DB")) {
+                    Logger.log(LogLevel.INFO, `Overriding macro ${macro.token} (${macro.replacement}) with ${cmdVars.DBname}`);
+                    macro.replacement = cmdVars.DBname;
```

Comment:
This might work for simpleui, I don't expect it to work for the paramsapp though? Have you tried this stuff with the params app?

Reply:
Checked the "Parameters" app instead of fleetviewer by accident. Fixed in last commit

Reply:
the "parameters app" *is* what you wanted to check, fleetviewer is a totally different repo

Reply:
Now I'm a little confused lol. There's four links on most of the systems that I've worked with: M1 graphs (fleetviewer), M2 graphs (fleetviewer), Parameters, and Event Log. So far changing the DB works for the M1 and M2 graph links. However, I need to be able to override the DB name in the Parameters application as well?

Reply:
I guess it's appropriate to fix the Parameters application, too.  I think that means that when there's an override in simpleui-server, you have to tweak the Parameters appLink to append `?database=VALUE", and also modify the php call inside Parameters.html , or post-process its results.

```
appLink.3.name    = Parameters
appLink.3.url     = /bms/html/ParamsApp/Parameters.html
appLink.3.tooltip = Start the Web UI Parameter Editor on BMS Server ${BMS_MYSQL_HOST}
```